### PR TITLE
adds axolotls

### PR DESCRIPTION
**Pull request by kawoppi**
## About The Pull Request
Axolotls have a chance of spawning roundstart at each moisture trap, just like their fellow amphibians. Their spawn chance is half that of a frog.
on floor

rest in peace
rip

You can pick them up and put them on your head as well.
on head

## Why It's Good For The Game
Rare animals can be a rewarding thing to find in maintenance without giving a combat/utility advantage over others. Also axolotls are cute.

## Changelog

:cl:
add: axolotls have taken refuge in the station's moisture traps
/:cl:

### CODE REVIEW

**Review by axietheaxolotl — APPROVED** [review]
YES.

File: code/game/objects/effects/spawners/random/trash.dm [hunk 1]
```diff
@@ -140,6 +140,7 @@
 		/obj/item/clothing/head/cone = 7,
 		/obj/item/clothing/suit/caution = 3,
 		/mob/living/simple_animal/hostile/retaliate/frog = 2,
+		/mob/living/simple_animal/axolotl = 1,
```
Comment (san7890) [inline code/game/objects/effects/spawners/random/trash.dm:143]:
```suggestion
		/mob/living/simple_animal/axolotl = 1,
		/mob/living/simple_animal/hostile/retaliate/frog = 2,
```

alphabetization is good

**Comment by epicgamer545** [comment]
This is the best PR since the mothroach

**Comment by Cheshify** [comment]
good pr

**Comment by Wallemations** [comment]
Big question
Can you scoop them up in buckets?

**Comment by xPokee** [comment]
Best thing since frogs

File: code/game/objects/effects/spawners/random/trash.dm [hunk 2]
```diff
@@ -139,6 +139,7 @@
 	loot = list( // This spawner will scatter water related items around a moist site.
 		/obj/item/clothing/head/cone = 7,
 		/obj/item/clothing/suit/caution = 3,
+		/mob/living/simple_animal/axolotl = 1,
```
Comment (JohnFulpWillard) [inline code/game/objects/effects/spawners/random/trash.dm:142]: These are organized by rarity I think, so axolotl would have to be last as it's the rarest.

File: code/modules/mob/living/simple_animal/friendly/axolotl.dm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	desc = "Quite the colorful amphibian!"
+	icon_state = "axolotl"
+	icon_living = "axolotl"
+	icon_dead = "axolotl_dead"
+	maxHealth = 10
+	health = 10
+	attack_verb_continuous = "nibbles"
+	attack_verb_simple = "nibble"
+	melee_damage_lower = 0
+	melee_damage_upper = 0 //their teeth are just for gripping food, not used for self defense nor even chewing
```
Comment (JohnFulpWillard) [inline code/modules/mob/living/simple_animal/friendly/axolotl.dm:12]: This is default on all simple animals, but there's a ``harm_intent_damage`` if ever it's needed.

**Comment by kawoppi** [comment]
> Big question Can you scoop them up in buckets?

Unfortunately you can't. It would be neat but I think storing them in buckets might break some things that would be awkward to fix. You can still put them in your pockets though.

**Comment by calliophis-maculiceps** [comment]
axolotls don't like being out of water

**Comment by warface1234455** [comment]
@kawoppi will you add in hand sprite

**Comment by kawoppi** [comment]
> @kawoppi will you add in hand sprite

I didn't add one because I thought I couldn't make one without it looking too cluttered. But now that I think about it it might be more doable than I thought. I think I could add some soon.

**Pull request merged**